dbms/src/Flash/Pipeline/Exec/PipelineExec.cpp

```diff
@@ -24,33 +24,39 @@ extern const char random_pipeline_model_execute_prefix_failpoint[];
 extern const char random_pipeline_model_execute_suffix_failpoint[];
 } // namespace FailPoints
 
-#define HANDLE_OP_STATUS(op, op_status, expect_status)                                            \
-    switch (op_status)                                                                            \
-    {                                                                                             \
-    case (expect_status):                                                                         \
-        break;                                                                                    \
-    /* For the io status, the operator needs to be filled in io_op for later use in executeIO. */ \
-    case OperatorStatus::IO:                                                                      \
-        assert(!io_op);                                                                           \
-        assert(op);                                                                               \
-        io_op.emplace((op).get());                                                                \
-    /* For unexpected status, an immediate return is required. */                                 \
-    default:                                                                                      \
-        return (op_status);                                                                       \
+#define HANDLE_OP_STATUS(op, op_status, expect_status)                                                 \
+    switch (op_status)                                                                                 \
+    {                                                                                                  \
+    case (expect_status):                                                                              \
+        break;                                                                                         \
```

Comment:
Why this case no return statement? A little weird.

Reply:
ok, comment `For the expected status, it will not return here, but instead return control to the macro caller, who will continue to call the next operator.` added.